internal/operator/operator_group_test.go

```diff
@@ -13,49 +13,54 @@ import (
 
 var _ = Describe("OperatorGroup", func() {
 	logger.InitLogger("debug")
+	var operatorClient operatorClient
+	var operatorGroupData OperatorGroupData
```

Comment:
Same as the namespace. Just leave the vars out of the BeforeEach. Everything else in.

Reply:
In fact, pretty much all of my comments there apply here.